Sources/NetworkService/ParametersEncoder/ParametersEncoder.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// A type that encodes URL parameters into a request.
+public protocol ParametersEncoder {
+    /// Encodes provided set of parameters as URL parameters into the given request's URL.
+    ///
+    /// - Parameters:
+    ///  - parameters: An optional set of URL parameters to be encoded.
+    ///  - request: A request object to which URL the encoded parameters will be assigned.
```

Comment:
Please check the formatting, the  doc comments don't display properly
<img width="574" alt="Screenshot 2023-08-23 at 09 56 44" src="https://github.com/ios-course/simple-network-service/assets/32869814/a791f0f9-0fa6-4adf-8bdb-8ba8332796b1">

```suggestion
    /// - Parameters:
    ///     - parameters: An optional set of URL parameters to be encoded.
    ///     - request: A request object to which URL the encoded parameters will be assigned.
```

Reply:
Fixed